finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceIT.java

```diff
@@ -72,12 +81,16 @@ public void testFallbackForGet() throws InterruptedException {
         int propertiesSizeFallBack = obj.size();
         assertTrue(propertiesSize > propertiesSizeFallBack, 
                    "The total number of properties from the @Fallback method "
-                 + "is not smaller than the number from the system service, as expected.");
+                 + "is not smaller than the number from the system service" 
+                +  "as expected.");
```

Comment:
put `"as expected.");` back to its original line (it's fine in the original line, it's not too long)

or 

Align the plus sign of `+  "as expected.");` with the plus sign in the previous line